base/shell/explorer/settings.cpp

```diff
@@ -73,6 +74,9 @@ BOOL TaskbarSettings::Load()
         if (sr.Position > ABE_BOTTOM)
             sr.Position = ABE_BOTTOM;
     }
+    
+    /* Policies Settings */
+    bPolicyNoSimpleStartMenu = SHRestricted(REST_NOSTARTPANEL);
```

Comment:
SHRestricted already handles caching (and will handle changes made to the registry without requiring an explorer restart), so don't save this value in explorer, just query it every time it's needed.